### ISSUE TITLE
Support of daemon mode with generic bluetooth controllers

### ISSUE THREAD

**Issue by petr666** [xboxdrv/xboxdrv#190]
Hi,
 it would be great if daemon mode has supported also bluetooth controllers. Since their connection/disconnection is detectable via udev I guess it should be possible? It would be very useful since bluetooth controllers go to sleep after few minutes of inactivity so daemon mode hotplugging feature will allow us to do reconnect without restarting of game in case controller goes to sleep.

**Comment by bitrevo** [comment]
Yes, Im troubleshooting for few hours on this issue and the daemon mode still not working with my bluetooth gamepad. I am using below command to lunch xboxdrv in daemon and enable the debugging.

`sudo /bin/xboxdrv -c /etc/xboxdrv.conf --daemon --dbus disabled --debug --detach-kernel-driver`

Seems like the XboxDrvDaemon only scan root devices of usb interface (same as `lsusb` command).
